invenio_rdm_records/resources/serializers/schemaorg/schema.py

```diff
@@ -0,0 +1,244 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+# Copyright (C) 2021 Northwestern University.
+# Copyright (C) 2023 Graz University of Technology.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Schemaorg based Schema for Invenio RDM Records."""
+from typing import Optional
+from edtf import parse_edtf
+from edtf.parser.grammar import ParseException
+from flask_resources.serializers import BaseSerializerSchema
+from pydash import py_
```

Comment:
I understand the need for a library like this, I am tired of nesting `.get().get()` 😂  

However, I am not sure where we are standing in the project on this subject. I will ping @zzacharo for this one

Reply:
I discussed this with @lnielsen a few weeks ago. But there is more than one way to do this, and I respect what general approach InvenioRDM takes on nested dicts.